### ISSUE TITLE
[BUG] gridplot got multiple values for keyword argument 'logo'

### ISSUE THREAD

**Issue by WheatMaThink** [bokeh/bokeh#13369]
### Software versions

Python version      :  3.9.13 (v3.9.13:6de2ca5339, May 17 2022, 11:23:25) 
IPython version     :  8.15.0
Tornado version     :  6.3.3
Bokeh version       :  3.2.2
BokehJS static path :  ../lib/python3.9/site-packages/bokeh/server/static
node.js version     :  v16.17.0
npm version         :  8.19.2
Operating system    :  macOS-10.15.7-x86_64-i386-64bit

### Expected behavior

...

### Observed behavior

File "/testBokeh/temp.py", line 35, in <module>
    grid = gridplot([[p1, p2], [p3, p4]], toolbar_options=dict(logo='gray'))
  File "site-packages/bokeh/layouts.py", line 333, in gridplot
    toolbar = Toolbar(
TypeError: bokeh.models.tools.Toolbar() got multiple values for keyword argument 'logo'

### Example code

```Python
from bokeh.layouts import gridplot
from bokeh.plotting import figure, show
import pandas as pd
import numpy as np

data = pd.DataFrame(np.random.randn(200, 4), columns=['A', 'B', 'C', 'D'])

p1 = figure()
p1.circle(data['A'], data['B'])

p2 = figure()
p2.triangle(data['C'], data['D'])

p3 = figure()
p3.line(data['A'], data['C'])

p4 = figure()
p4.line(data['B'], data['D'])

grid = gridplot([[p1, p2], [p3, p4]], toolbar_options=dict(logo='gray'))
# grid = gridplot([[p1, p2], [p3, p4]])
show(grid)
```

**Comment by jwlondon98** [comment]
I'm still getting this issue in Bokeh version 3.2.2, Python 3.9.5

**Comment by bryevdv** [comment]
@jwlondon98 This was fixed in #13370 as indicated above. Issues are closed whenever a fix PR is merged to the latest development branch. An issue being closed does not mean there has been a new release to include the fix already (as is the case here&mdash;3.3 is not out yet but is expected relatively soon)

**Comment by AKorets** [comment]
temporary solution
pip install bokeh==3.2.0
